src/dailyai/services/daily_transport_service.py

```diff
@@ -249,8 +249,9 @@ def on_transcription_message(self, message: dict):
                 participantId = message["participantId"]
             elif "session_id" in message:
                 participantId = message["session_id"]
-            frame = TranscriptionQueueFrame(message["text"], participantId, message["timestamp"])
-            asyncio.run_coroutine_threadsafe(self.receive_queue.put(frame), self._loop)
+            if self._my_participant_id and participantId != self._my_participant_id:
```

Comment:
I thought we'd filter these out in the LLMResponseAggregators -- is it better to do it here? The only downside I can think of is if people wanted to do something (I don't know what) with the transcriptions that come from audio frames we send.

Reply:
I'm in favor of doing it here, because we go to a fair amount of trouble to use the LLM's output as a pseudo-transcript later. We can revisit it for other use cases if they appear, but this simplifies some of the downstream code